gobblin-data-management/src/main/java/org/apache/gobblin/data/management/conversion/hive/query/HiveAvroORCQueryGenerator.java

```diff
@@ -160,21 +163,25 @@ public static String generateCreateTableDDL(Schema schema,
     //    (evolution does not matter if its new destination table)
     // 4. If evolution is disabled, and destination table does exists
     //    .. use columns from destination schema
+    // Make sure the schema attribute will be updated in source-of-truth attribute.
+    // Or fall back to default attribute-pair used in Hive for ORC format.
+    if (tableProperties.containsKey(SCHEMA_SOURCE_OF_TRUTH)) {
+      tableProperties.setProperty(tableProperties.getProperty(SCHEMA_SOURCE_OF_TRUTH), schema.toString());
+      tableProperties.remove(SCHEMA_SOURCE_OF_TRUTH);
```

Comment:
Just curious about what is this for?

Reply:
So, usually this should be `avro.schema.literal`, which is what we did internally for most of tables. 

While there are cases where we cannot set `avro.schema.literal`: When there were older files  written in lower-case ORC but newly added `avro.schema.literal` is case-preserved.  Any mismatch between `avro.schema.literal` and file-level schema will cause problem, if `avro.schema.literal` is being set which gives offline system like Dali an illusion that this is the correct schema. 
But we still need something to inform pipelines like "view-creation" the Avro schema of a table, since interfering it from Hive schema is lossy.  When this case happens, we will set this `SCHEMA_SOURCE_OF_TRUTH ` to be something else (different from `avro.schema.literal`) as a workaround to provide Avro schema but not messing with Dali.